RecoParticleFlow/PFClusterProducer/python/particleFlowRealisticSimClusterHGCCalibrations_cfi.py

```diff
@@ -2,5 +2,5 @@
 
 minEtaCorrection = cms.double(1.4)
 maxEtaCorrection = cms.double(3.0)
-hadronCorrections = cms.vdouble(1.24, 1.24, 1.24, 1.23, 1.24, 1.25, 1.29, 1.29)  
-egammaCorrections = cms.vdouble(1.00, 1.00, 1.01, 1.01, 1.02, 1.03, 1.04, 1.04)  
+hadronCorrections = cms.vdouble(1.28, 1.28, 1.24, 1.19, 1.17, 1.17, 1.17, 1.17)  
```

Comment:
if these calibrations are only appropriate for the V9 geometry, the values should be modified using the `phase2_hgcalV9` modifier, so the old versions of HGCal will not get incorrect calibrations